scripts/templates/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # {{templateWarning}}
-FROM quay.io/operator-framework/helm-operator:v1.9.1
+FROM quay.io/operator-framework/helm-operator:v1.18.0
```

Comment:
🐛 upgrade to a newer base. this was done previously or one instance of the operator, but it was missing from the template, so it wasn't permanent for the next version.